extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -150,32 +167,34 @@
 }
 
 .i-amphtml-story-interactive-slider-input::-webkit-slider-thumb {
+  transform: rotate(45deg) !important;
   -webkit-appearance: none !important;
   width: var(--thumb-size) !important;
   height: var(--thumb-size) !important;
   background: var(--interactive-accent-color) !important;
   border-radius: 50% !important;
   position: relative !important;
+  transition: border-radius 0.5s, box-shadow 0.5s !important;
```

Comment:
Please add `var(--i-amphtml-interactive-ease-out-curve)` to both of these transitions